firmware/tunerstudio/rusefi.input

```diff
@@ -1113,7 +1118,7 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 		field = "maxRPM minTPS", tChargeMaxRpmMinTps
 		field = "maxRPM maxTPS", tChargeMaxRpmMaxTps
 
-	dialog = baseInjection, "Injection"
+	dialog = baseInjection, "General"
```

Comment:
Now this is not a separate dialog, but a panel inside 'injection settings' dialog